app/src/main/java/net/bible/android/control/page/PageControl.java

```diff
@@ -111,6 +111,20 @@ public void shareVerse(VerseRange verseRange) {
 			Dialogs.getInstance().showErrorMsg("Error sharing verse");
 		}
 	}
+
+	private String getCopyShareText(Book book, VerseRange verseRange) {
+		try {
+			String referenceName = StringUtils.replace(verseRange.getName(),
+					BibleNames.instance().getPreferredName(verseRange.getStart().getBook()),
+					BibleNames.instance().getPreferredNameInLocale(
+							verseRange.getStart().getBook(), new Locale(book.getLanguage().getCode()))
+			);
+			return referenceName+"\n"+"\n"+swordContentFacade.getTextWithVerseNumbers(book, verseRange);
+		} catch (Exception e) {
+			Log.e(TAG, "Error converting verse from OSIS to text.", e);
+			return null;
+		}
+	}
```

Comment:
Hmmh. It looks like we need to tweak VerseRange if we are getting reference string directly via getName(). I.e. create there verseRange.getLocalizedName(). I don't like search & replace approach.

Reply:
So this needs tweaking jsword. I can also do this at some point.